### ISSUE TITLE
Fails to detect anything when epoch number is large

### ISSUE THREAD

**Issue by ryanaleksander** [thtrieu/darktf#725]
Hello, I'm having a very weird problem. At first, I trained a new model to detect 3 object classes using yolo.cfg as base. The model were trained with standard parameters in 150 epochs. Then I trained another model, using the same parameters and dataset as the first one (3100 images), but this time with 1000 epochs.

The first model has ~30% mAP. The second model can't detect anything, in any images, even those in the training set. Shouldn't the second model detect better, at least on training dataset, since it was probably overfitted?

**Comment by rij12** [comment]
Sorry to hijack your question but how did you find the mAP?

**Comment by ryanaleksander** [comment]
@rij12 I used this: https://github.com/Cartucho/mAP

**Comment by Jped** [comment]
Hi did you ever figure out the answer to this question? I was able to train a tiny-yolo model for 300 epochs and got the loss down to around 1e-9, but when I test the model on the training set I get a 0% mAP.

**Comment by CRonaldo1997** [comment]
same problem, any one solved it?

**Comment by CRonaldo1997** [comment]
config the confidence threshold to a lower value, eg:0.1
solved my problem.
